app/src/main/java/ch/epfl/sdp/musiconnect/Band.java

```diff
@@ -1,39 +1,26 @@
 package ch.epfl.sdp.musiconnect;
 
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-
 import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.Set;
-import java.util.HashSet;
 
 /**
  * @author Manuel Pellegrini, EPFL
  */
-public class Band extends User implements Performer {
+public class Band extends User {
 
     private String bandName;
-    private Musician leader;
-    private Set<Musician> members;
-    private String videoURL;
+    private String leader;
     private ArrayList<String> musicianEmails;
 
     private static final int MAX_BAND_NAME_LENGTH = 16;
-    private static final int MAX_VIDEO_URL_LENGTH = 2048;
 
 
-    public Band(String bandName, Musician leader) {
+    public Band(String bandName, String leader) {
         super();
         setBandName(bandName);
-        members = new HashSet<Musician>();
         events = new ArrayList<>();
+        musicianEmails = new ArrayList<>();
```

Comment:
ArrayList<String> would be better I think

Reply:
There are yet defined as List<String>